src/utils/fetchTautulli.ts

```diff
@@ -90,23 +93,64 @@ export async function getLibraries(
   excludeInactive = true,
 ): Promise<TautulliLibrary[]> {
   const settings = getSettings()
+  const libraries = (await fetchTautulli<TautulliLibrary[]>('get_libraries'))
+    ?.response?.data
+  if (libraries == null) {
```

Comment:
Any reason you changed this to a null check instead of the previous falsy check? Considering you're using abstract equality comparison instead of strict, makes me wonder even more.